prime-router/metadata/hl7_mapping/ORU_R01/base/message-header.yml

```diff
@@ -1,13 +1,13 @@
 name: ORU-R01-message-header
 elements:
   - name: sending-application-namespace-id
-    resource: '%resource.source.extension(%`rsext-source-software-vendor-org`).value.resolve()'
-    value: [ '%resource.extension(%`rsext-assigning-authority-namespace-id`).value' ]
+    # Original value is in %resource.extension(%`rsext-assigning-authority-namespace-id`).value
+    value: [ '''CDC PRIME - Atlanta, Georgia (Dekalb)''' ]
```

Comment:
Why are we hard coding?

Reply:
This is what is done in ReportStream today.  It tells the receivers the data is coming from ReportStream instead of from an entity directly.